textproc/glow/Portfile

```diff
@@ -30,10 +30,10 @@ build.args-append   -ldflags=\"-X 'main.Version=${version}'\"
 destroot {
     xinstall -m 0755 ${worksrcpath}/${name} ${destroot}${prefix}/bin/
     # All supported shells: bash, fish, powershell, zsh
+    # completion for fish included in fish 4.1
```

Comment:
shouldn't this instead be: `# completion for glow included in fish 4.1`?

Reply:
Updated